themes-default/slim/src/router.js

```diff
@@ -297,8 +300,8 @@ const changesRoute = {
 };
 
 const ircRoute = {
-    path: '/IRC',
-    name: 'IRC',
+    path: '/irc',
```

Comment:
Note that this will break the IRC page link. The link is case sensitive, so if you want to change it, change the route as well.
https://github.com/pymedusa/Medusa/blob/98aeff745c8338ebaa4fdd433adb801031c10542/medusa/server/web/home/irc.py#L12